src/python/pants/backend/python/macros/caof_utils_test.py

```diff
@@ -23,26 +60,30 @@ def test_flatten_overrides_to_dependency_field() -> None:
 
 
 def test_flatten_overrides_same_key() -> None:
-    # Invalid to specify same key multiple times.
-    with pytest.raises(InvalidFieldException):
+    with pytest.raises(InvalidFieldException, match="Conflicting overrides"):
         flatten_overrides_to_dependency_field(
             {"d1": {"dependencies": []}, ("d1", "d2"): {"dependencies": []}},
             macro_name="macro",
             build_file_dir="dir",
         )
 
+    with pytest.raises(InvalidFieldException, match="Conflicting overrides"):
+        flatten_overrides_to_dependency_field(
+            {("d1", "d1"): {"dependencies": []}},
+            macro_name="macro",
+            build_file_dir="dir",
+        )
+
 
 def test_flatten_overrides_only_dependencies_field() -> None:
-    # Invalid to specify same key multiple times.
```

Comment:
This comment threw me off because it's copy-pasta. Changed the tests to not need comments by using `match=`